cloudify_cli/cli/helptexts.py

```diff
@@ -244,3 +244,12 @@
                     'skips the first resource [default: 0]'
 DRY_RUN = 'If set, no actual operations will be performed. This ' \
           'only prints the executed tasks, without side effects'
+
+MANAGER_IP = 'The private IP of the new Cloudify-Manger'
```

Comment:
Maybe add a little more info re which IP to use (e.g. "this is the IP through which the agents connect to the manager"